composites/AlgoliaSearch/SearchBar.js

```diff
@@ -7,43 +7,74 @@ import debounce from "lodash/debounce";
 import colors from "../../style-guide/colors.json";
 import SearchIcon from "../../style-guide/svg/search.svg";
 import { Icon } from "../Plugin/Shared/components/Icon";
+import { YoastButton } from "../Plugin/Shared/components/YoastButton";
+import breakpoints from "../../style-guide/responsive-breakpoints.json";
 
 const messages = defineMessages( {
 	headingText: {
 		id: "searchBar.headingText",
 		defaultMessage: "Search the Yoast knowledge base",
 	},
+	placeholderText: {
+		id: "searchBar.placeholderText",
+		defaultMessage: "Search the knowledge base",
+	},
+	buttonText: {
+		id: "searchBar.buttonText",
+		defaultMessage: "Search",
+	},
 } );
 
-const SearchLabel = styled.label`
-	width: 2em;
-	height: 2em;
-	float: left;
-	margin-top: 0.25em;
-`;
-
 const SearchBarWrapper = styled.div`
-	overflow: hidden;
-	width: 100%;
 	padding: 0 16px;
-	box-sizing: border-box;
+
+	form {
+		display: flex;
+
+		@media screen and (max-width: ${ breakpoints.mobile } ) {
+			flex-wrap: wrap;
+		}
+	}
+
+	@media screen and (max-width: ${ breakpoints.mobile } ) {
+		button {
+			min-width: 100%;
+			margin-top: 1em;
+		}
+	}
+`;
+
+const SearchHeading = styled.h2`
+	font-size: 1em;
+	margin: 0.5em 0 0.5em 58px;
+
+	@media screen and (max-width: ${ breakpoints.mobile } ) {
+		margin-left: 0;
+	}
+`;
+
+const SearchLabel = styled.label`
+	flex: 0 0 42px;
+	height: 3em;
+	// it's already a flex item, let's make it also a flex container to align the svg icon
+	display: inline-flex;
+	align-items: center;
 `;
 
 const SearchBarInput = styled.input`
+	flex: 1 1 auto;
 	box-sizing: border-box;
-	height: 2.5em;
-	width: calc(100% - 2em);
+	height: 3em;
 	box-shadow: inset 0 2px 8px 0px rgba(0,0,0,0.3);
```

Comment:
Not your code, but if possible, add spaces

Reply:
I'm not sure we have standards for this :) this is not JavaScript, it's CSS. Anyways, I'll update all the occurrences across all files, there are a few...